pkg/notifier/notifier.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notifier
```

Comment:
nit: I'd suggest to name this package e.g. `kubeletnotifier` as it's very kubelet-specific, not any generic notifier. I could also consider moving this under `pkg/nfd-topology-updater` and starting it from the TopologyUpdater, similar to the resourcemonitor/resourcescanner. WDYT?

Reply:
Done. Thanks